src/components/table/virtualized-table.tsx

```diff
@@ -193,7 +205,7 @@ export function VirtualizedTable<T extends object>({
         orderedRows={orderedRows}
         innerRef={innerRef}
         outerRef={outerRef}
-        onItemsRendered={onItemsRendered}
+        onItemsRendered={itemsRenderHandler}
```

Comment:
i would name it `handleItemsRendered` (replacing `on` with `handle`), but we don't have any convention so i'm ok with that. We can someday discuss if we want to follow rules like here: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md